THCalendarDatePickerExample/THCalendarDatePickerExample/TestViewController.m

```diff
@@ -30,15 +30,14 @@ -(void)refreshTitle {
 - (IBAction)touchedButton:(id)sender {
     if(!self.datePicker)
         self.datePicker = [THDatePickerViewController datePicker];
-    self.datePicker.date = self.curDate;
```

Comment:
The order here is not wrong, but as I said in #74 it should be used like that, but can be used later too...